src/shared/sources/my/index.js

```diff
@@ -0,0 +1,73 @@
+const assert = require('assert')
+const maintainers = require('../_lib/maintainers.js')
+const parse = require('../_lib/parse.js')
+
+const country = 'iso1:MY'
+
+const schemaKeysByHeadingFragment = {
+  'positif': 'cases',
+  'negatif': 'testedNegative',
+  'Pulih': 'recovered',
+  'Rawatan Rapi': 'icu',
+  'Kematian': 'deaths',
+}
```

Comment:
Mapping was grabbed from the english version of the site, which you can change to client-side.